public/app/features/dashboard-scene/panel-edit/VizPanelManager.tsx

```diff
@@ -74,6 +76,7 @@ export class VizPanelManager extends SceneObjectBase<VizPanelManagerState> {
       panel: sourcePanel.clone({ $data: undefined }),
       $data: sourcePanel.state.$data?.clone(),
       sourcePanel: sourcePanel.getRef(),
+      libraryPanel: getLibraryPanel(sourcePanel)?.clone(),
```

Comment:
After merging #83077 the viz will not display if you hit refresh in the edit view. If `panel: sourcePanel.clone({ $data: undefined }),` is changed to `panel: sourcePanel.clone(),` IE the panel `$data` is kept intact, it works.

I could not figure out why.

Reply:
It worked fine for me as well

Reply:
It was fixed on main :)